docs/workflow/docs.md

```diff
@@ -0,0 +1,68 @@
+# Documentation
+
+This guide is for developers who wish to contribute to the documentation for our codebases. Sometimes you would like to test your doc changes and see them directly on your local studio; this goes through some of methods that are available.
+
```

Comment:
I'll probably add my own `mini-section` here to detail about the working of our documentation registry/virtual assistant